hacking/build_library/build_ansible/announce.py

```diff
@@ -1,293 +0,0 @@
-# coding: utf-8
-# Copyright: (c) 2019, Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-# Make coding more python3-ish
-from __future__ import (absolute_import, division, print_function)
-__metaclass__ = type
-
-
-import asyncio
-import datetime
-import hashlib
-
-import aiohttp
```

Comment:
@mattclay do we also need to clean up some deps?

Reply:
It doesn't look like it, since `aiohttp` isn't listed in any of our direct requirements.